lunatrace/bsl/frontend/src/pages/vulnerabilities/detail/EquivalentVulnerabilitiesList.tsx

```diff
@@ -40,6 +42,16 @@ export const AffectedPackagesList: React.FC<{ relatedVulns: VulnInfoDetails['equ
                 {relatedVulns.map(({ equivalent_vulnerability: relatedVuln }) => {
                   const severity = getCvssVectorFromSeverities(relatedVuln.severities);
 
+                  if (!relatedVuln.last_fetched) {
```

Comment:
Should we just hide vulns in this case? I think vuln ingestion is an implementation detail we should not expose to the user.